test/unit/cli-domain-package-components.js

```diff
@@ -6,96 +6,146 @@ var path = require('path');
 var sinon = require('sinon');
 var _ = require('underscore');
 
-var initialise = function(){
-
-  var fsMock = {
-    existsSync: sinon.stub(),
-    lstatSync: sinon.stub(),
-    mkdirSync: sinon.spy(),
-    readdirSync: sinon.stub(),
-    readFileSync: sinon.stub(),
-    readJson: sinon.stub(),
-    readJsonSync: sinon.stub(),
-    writeFile: sinon.stub().yields(null, 'ok'),
-    writeJson: sinon.stub().yields(null, 'ok')
-  };
-
-  var pathMock = {
-    extname: path.extname,
-    join: path.join,
-    resolve: function(){
-      return _.toArray(arguments).join('/');
-    }
-  };
-
-  var Local = injectr('../../src/cli/domain/package-components.js', {
-    'fs-extra': fsMock,
-    'uglify-js': {
-      minify: function(code){
-        return {
-          code: code
-        };
+describe('cli : domain : package-components', function(){
+
+  var packageStaticFilesStub;
+
+  var initialise = function(){
+
+    var fsMock = {
+      existsSync: sinon.stub(),
+      lstatSync: sinon.stub(),
+      mkdirSync: sinon.spy(),
+      readdirSync: sinon.stub(),
+      readFileSync: sinon.stub(),
+      readJson: sinon.stub(),
+      readJsonSync: sinon.stub(),
+      writeFile: sinon.stub().yields(null, 'ok'),
+      writeJson: sinon.stub().yields(null, 'ok')
+    };
+
+    var pathMock = {
+      extname: path.extname,
+      join: path.join,
+      resolve: function(){
+        return _.toArray(arguments).join('/');
       }
-    },
-    path: pathMock,
-    './package-static-files': sinon.stub().yields(null, 'ok'),
-    './package-template': sinon.stub().yields(null, { type: 'jade', src: 'template.js', hashKey: '123456'})
-  }, { __dirname: '' });
+    };
```

Comment:
I see a lot of `arrange` before running this unit test. maybe this component does too many things?